src/server/server_family.cc

```diff
@@ -522,6 +516,247 @@ bool IsReplicatingNoOne(string_view host, string_view port) {
   return absl::EqualsIgnoreCase(host, "no") && absl::EqualsIgnoreCase(port, "one");
 }
 
+struct SaveStagesInputs {
```

Comment:
according to the stylieguide, consider make it a class and declare variables as protected (they have underscores).

Reply:
I moved this out to a separate struct only to use aggregate initialization, its not possible with private/protected members. Alternatively I can just move the fields back and make a separate gigantic contructor that accepts them all

Reply:
Ok, how about the option of making the fields public (without underscores) and make this struct a data member instead of an inheritance?

Reply:
will open a separate fix on this 😬